itest.sh

```diff
@@ -52,6 +52,11 @@ tronfig -n MASTER /work/example-cluster/tronfig/MASTER.yaml
 tronfig /work/example-cluster/tronfig/MASTER.yaml
 cat /work/example-cluster/tronfig/MASTER.yaml | tronfig -n MASTER -
 
+if test -L /opt/venvs/tron/lib/python3.6/encodings/punycode.py; then
+    echo "Whoa, the tron package shouldn't have an encoding symlink!"
+    exit 1
```

Comment:
Can you link to the relevant ticket? I have a feeling the reasoning behind this might get lost otherwise.

Reply:
fixed